### DISCUSSION TITLE
Symbolic Link

### DISCUSSION THREAD

**Discussion by raine-works** [louislam/dockge#374] [Ask for Help] [answered]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

I moved all my homelab projects into a mono repo and symlinked individual packages into /opt/stacks and now I am no longer able to manage these stacks in Dockge. It shows the stacks as active but does not allow me to manage them.

### 📝 Error Message(s) or Log

This stack is not managed by Dockge.

### 🐻 Dockge Version

1.4.1

### 💻 Operating System and Arch

22.4.3

### 🌐 Browser

Google Chrome 120.0.6099.217

### 🐋 Docker Version

Docker 24.0.7

**Comment by louislam** [comment]
Should be same issue: https://github.com/louislam/dockge/discussions/373

**Comment by raine-works** [comment, marked as answer]
Using rsync -a instead of ln -s solved the issue for me.
